dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/DataIntegrity.ts

```diff
@@ -100,35 +102,58 @@ describeIf(
         .should("contain", formatPercentNumber(testData.insuranceKpis!.taxonomyEligibleNonLifeInsuranceActivities));
     }
 
-    /*
-    it("Create an Eu Taxonomy Financial dataset via upload form with all financial company types selected to assure " +
-        "that the upload form works fine with all options", () => {
-      const testData = getPreparedFixture("credit-institution-single-field-submission");
-      uploadCompanyViaApiAndEuTaxonomyDataForFinancialsViaFormAndVisitFrameworkDataViewPage(
+    function checkCreditInstitutionValues(
+      testData: EuTaxonomyDataForFinancials,
+      individualFieldSubmission: boolean,
+      dualFieldSubmission: boolean
+    ): void {
+      checkCommonFields("CreditInstitution", testData.eligibilityKpis!.CreditInstitution);
+      if (individualFieldSubmission) {
+        cy.get('div[name="tradingPortfolio"]')
+          .should("contain", "Trading portfolio")
+          .should("contain", formatPercentNumber(testData.creditInstitutionKpis!.tradingPortfolio));
+        cy.get('div[name="onDemandInterbankLoans"]')
+          .should("contain", "On demand interbank loans")
+          .should("contain", formatPercentNumber(testData.creditInstitutionKpis!.interbankLoans));
+        if (!dualFieldSubmission) {
+          cy.get("body").should("not.contain", "Trading portfolio & on demand interbank loans");
+        }
+      }
+      if (dualFieldSubmission) {
+        cy.get('div[name="tradingPortfolioAndOnDemandInterbankLoans"]')
+          .should("contain", "Trading portfolio & on demand interbank loans")
+          .should("contain", formatPercentNumber(testData.creditInstitutionKpis!.tradingPortfolioAndInterbankLoans));
+        if (!individualFieldSubmission) {
+          cy.get("body").should("not.contain", /^Trading portfolio$/);
+          cy.get("body").should("not.contain", "On demand interbank loans");
+        }
+      }
+    }
+
+    it(
+      "Create an Eu Taxonomy Financial dataset via upload form with all financial company types selected to assure " +
+        "that the upload form works fine with all options",
+      () => {
+        const testData = getPreparedFixture("company-for-all-types");
+        uploadCompanyViaApiAndEuTaxonomyDataForFinancialsViaFormAndVisitFrameworkDataViewPage(
           testData.companyInformation,
           testData.t
-      );
-      checkCommonFields("CreditInstitution", testData.t.eligibilityKpis!.CreditInstitution);
-      cy.get('div[name="tradingPortfolioAndOnDemandInterbankLoans"]')
-          .should("contain", "Trading portfolio & on demand interbank loans")
-          .should("contain", formatPercentNumber(testData.t.creditInstitutionKpis!.tradingPortfolioAndInterbankLoans));
-      cy.get("body").should("not.contain", /^Trading portfolio$/);
```

Comment:
These two lines, checking the body's content, did not get readded. What exactly is their purpose.